test/java/org/opendatakit/briefcase/model/FormStatusTest.java

```diff
@@ -43,4 +45,14 @@ public void knows_how_to_build_paths_to_assets_inside_the_storage_directory() {
     );
   }
 
+  @Test
+  public void gets_manifest_url_from_remoteform() {
+    FormStatus form = new FormStatus(null);
+    String supposedManifestUrl = "manifest/test/url";
```

Comment:
Small language tip: Use `expected` instead of `supposed` as in `expectedManifestUrl`.